ovs/extensions/plugins/asdmanager.py

```diff
@@ -194,20 +203,38 @@ def delete_asd(self, disk_id, asd_id):
         """
         return self._call(requests.post, 'disks/{0}/asds/{1}/delete'.format(disk_id, asd_id), timeout=60)
 
-    def get_update_information(self):
+    def list_asd_services(self):
+        """
+        Retrieve the ASD service names and their currently running version
         """
-        Checks whether update for openvstorage-sdm package is available
+        return self._call(requests.get, 'asds/services', timeout=60, clean=True)['services']
+
+    def get_package_information(self):
+        """
+        Retrieve the package information for this ALBA node
         :return: Latest available version and services which require a restart
         """
-        return self._call(requests.get, 'update/information', timeout=120, clean=True)
+        # For backwards compatibility we first attempt to retrieve using the newest API
+        try:
+            return self._call(requests.get, 'update/package_information', timeout=120, clean=True)
+        except NotFoundError:
+            update_info = self._call(requests.get, 'update/information', timeout=120, clean=True)
+            return {'alba': [{'candidate': update_info['version'],
+                              'installed': update_info['installed'],
+                              'name': 'openvstorage-sdm',
+                              'namespace': 'alba',
+                              'services_to_restart': []}]}
 
     def execute_update(self, status):
```

Comment:
Why is the status still passed in? It's not relevant anymore for the new logic right? I'd expect that you hardcode ```start``` (or whatever it is to trigger an update on the old sdm) to the old API, so it's triggered, and as soon as it's restarted you'll be able to track progress using the new APIs.